fortls/__init__.py

```diff
@@ -6,13 +6,16 @@
 import sys
 from multiprocessing import freeze_support
 
-from ._version import __version__
 from .helper_functions import resolve_globs, only_dirs
 from .jsonrpc import JSONRPC2Connection, ReadWriter, path_from_uri
 from .langserver import LangServer
 from .parse_fortran import fortran_file, process_file
 from .interface import commandline_args
 
+from .version import __version__
```

Comment:
sort imports (I can do that at the very end)

Reply:
Not sure how you want them sorted? `isort` makes a _lot_ of changes, so probably not that?

Reply:
It's fine. I just saw that I was not running isort as a precommit hook. I will fix the sorting in a different PR to keep this one minimal.